app/femr/business/services/core/IEncounterService.java

```diff
@@ -74,7 +74,7 @@
 
     /**
      * Creates a bunch of tab fields that belong to a chief complaint
-     *
+     * Method is returning all none deleted encounters
```

Comment:
I don't think the comment is necessary on this method because it's only for creating new tab fields. It should be more explicitly commented in methods which are retrieving a patient encounter by id so the developer knows they aren't getting deleted encounters back